DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>Explain (or link to a reference) what "boostraping" means. </li><li>Explain how this method differes, in principle, from the error calculation that you did above. When should you use one or the other? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<ol><li>adjusted</li><li>adjusted</li></ol>